### ISSUE TITLE
FR: Reapack distribution

### ISSUE THREAD

**Issue by X-Raym** [ebu/ear-production-suite#8]
Hi !

I don't use ADM in my workflow but I want to thank you for bringing to the reaper community. Awesome !

--
It would be interesting if you could share this pack via [Reapack](http://reapack.com/), reaper extension,
so it could be downloaded and install, and even updated right from REAPER, without having to manually files at different places.

It is meant for sharing reaper resources (scripts, extension, themes), and it can also share VST plugins in Resource /FX folder, so that REAPER will detect automatically.

It is pretty simple to set up so it definitely worth it. You can ask for [cfillion ](https://forum.cockos.com/showthread.php?t=177978) for help if needed :P

Thanks for your attention !

**Comment by musicbynumbers** [comment]
Agreed. This would be a great idea and make the process more efficient.

And thanks for making a fantastic production suite!

**Comment by plush2** [comment]
I agree as well. This suite is great and a simpler installation process would make integrating into one's workflow in Reaper easier.

**Comment by davidkamp** [comment]
This would be great!

**Comment by firthm01** [comment]
Closing as there is now an installation executable to resolve this. IIRC there were some blockers when we looked in to doing this as a reapack.
